.travis.yml

```diff
@@ -2,16 +2,19 @@ language: python
 sudo: false
 python:
     - 2.7
-    - pypy-5.6.0
+    - pypy
```

Comment:
Do we want to add pypy3?

Reply:
Let's try it, in #6 the interpreter crashed during the tests.

Reply:
Okay, let's drop it then.

pypy3 is alpha for a reason, and "Segmentation fault" could very well be that reason.